jamaljsr/polar
# feat(mining): automatically mine new blocks

Closes [#566](https://github.com/jamaljsr/polar/issues/566)

Based on the code of @zackypick 
Continues the unmerged PR [#613 ](https://github.com/jamaljsr/polar/pull/613)

### Description
Added auto mine dropdown button with several options (Off, every 30s, every 1m / 5m / 10m)
Button includes embedded progress bar (time left to mine)

### Steps to Test
1. Click the new toolbar button for options
2. Select any of the interval options
3. Auto mine will start according to interval
4. Verify that the blockheight changes when the progress bar goes to zero
5. Change the interval on the fly, and verify, that the mining still works
6. Select Off to turn down automine
